### ISSUE TITLE
“unable to execute line 88. Out of memory”

### ISSUE THREAD

**Issue by Lulu-Gao** [airbnb/lottie-web#2338]
Hello! Please help! :) I've been trying to export a .json for two days now and nothing I've tried has fixed the issue...
*disclaimer...I'm pretty new to AE and this is my first time using Bodymovin so something could be totally wrong in my process haha but I tried to research and find solutions but I'm really 100% stuck now.

Browser and Browser Version: iOS 10.14.6 Bodymovin v5.7.3
Storage: 16GB RAM, 26GB flash storage, emptied cache, cache folder on an external hard drive with 1.05TB space left
After Effects Version: 2020

**What did you do? Please explain the steps you took before you encountered the problem.**
I'm trying to get a .json file for a website. I'm using a .jpg sequence. I put all of the jpegs in a sequence and started the bodymovin extension. I've enabled scripts to write files and access network. I downloaded bodymovin first from Adobe Exchange and then uninstalled that and downloaded it from aescripts + aeplugins. Neither worked.

**What did you expect to happen?**
I expected it to export the .json file rather quickly and successfully, however it has done neither of those things.

**What actually happened?**
At first, it was taking 40 seconds to render each image, and it finished everything after 3 hours but then I got the line 88 error. I then emptied my cache, changed the cache folder to be hosted on my external hard drive, and selected "Skip Images Export" and "Copy original Images" as seen below. This made the images export faster (about 1 per second) but I still keep getting an "unable to execute line 88. out of memory" error after all the exports finish. Then the .json file just never finished saving and doesn't appear in the destination folder. 
*note: The first time I tried bodymovin it created a .json file but when I put that file in webflow it was blank...
<img width="638" alt="Screen Shot 2020-10-21 at 10 26 48 PM" src="https://user-images.githubusercontent.com/68263033/96819243-25f82e80-13f1-11eb-804c-a4c2637de1a8.png">

**Please provide a download link to the After Effects file that demonstrates the problem.**
https://drive.google.com/file/d/1k9sKl-HMIw6vdmD76hRHSo14WEfsK57_/view?usp=sharing

**Comment by bodymovin** [comment]
Hi, requested access to the folder, once you provide it, can you post in this thread so I can follow up?

**Comment by Lulu-Gao** [comment]
I think you should have access now! Please let me know if it didn’t work.

**Comment by bodymovin** [comment]
Hi, it says it's a 1.6Gb zip file. Are all those images?

**Comment by Lulu-Gao** [comment]
Yes, I believe so. Do I need to bring the file size down? And if so do you know by how much? Or do you know of a better way to make a high-resolution gif for a webflow website? We wanted something like what this page has: https://hopin.to/

**Comment by Lulu-Gao** [comment]
@bodymovin Have you found a solution or any other insights to the issue?

**Comment by bodymovin** [comment]
I don't think that type of scenario fits well with lottie. Can't you use a regular video?

**Comment by Lulu-Gao** [comment]
hmmm and make it auto-play and auto-loop on the website?

**Comment by bodymovin** [comment]
don't know about autoloop but videos can autoplay with the appropiate html attribute

**Comment by Lulu-Gao** [comment]
Okay, thank you! Do you know what went wrong with the bodymovin export though?

**Comment by bodymovin** [comment]
I guess so many images can't be buffered or there is not enough room in your disk

**Comment by Lulu-Gao** [comment]
I was using a 2TB external drive as my scratch disk so it might be the number of images that's causing the problem. Thanks though!
